src/common/eeprom_loadsave.h

```diff
@@ -0,0 +1,28 @@
+// eeprom_loadsave.h
+
+#ifndef _EEPROM_LOADSAVE_H
+#define _EEPROM_LOADSAVE_H
+
+#include "eeprom.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif //__cplusplus
+
+// load binary image (0x0000..0x0800) from usb flash (fatfs file)
+extern int eeprom_load_bin_from_usb(const char *fn);
+
+// save binary image (0x0000..0x0800) to usb flash (fatfs file)
+extern int eeprom_save_bin_to_usb(const char *fn);
+
+// TODO:
+extern int eeprom_load_xml_from_usb(const char *fn);
+
+// save xml file with all variables to usb flash (fatfs file)
+extern int eeprom_save_xml_to_usb(const char *fn);
```

Comment:
bool is not defined in C. Possible way is include <stdbool.h>, but I prefer usage of basic data types (int). There are many conventions for return values. when the logic is 1/0, for me is more transparent use value 1 for succes and value 0 for fail.

Reply:
serialization is not finished, I will change the suffix later. Maybe better suffix will be "_ffile" because fatfs functions are prefixed with "f_" (e.g. "f_open" instead of "open").

Reply:
`bool` is part of C specification for over 20 years now (ISO/IEC 9899), so I believe it is very much part of the language.

I am aware there are many conventions for return values, and that is the reason why I suggested using bool. When using `int`, the reader can't be sure what convention the API is using. However, when using `bool`, there is no dispute about it.

Reply:
bool is not supported in C, only with header stdbool.h ("bool", "true" and "false" are macros). I have different opinion to return values, i am not using bool datatype in C code and i will not change this. In future these functions should return system error codes.